### PR TITLE
Updated the bundle to use http-cache 2.5

### PR DESCRIPTION
**Pull request by Toflar**

### CODE REVIEW

**Comment by dbu** [comment]
we need to drop the test with lts 2: https://github.com/FriendsOfSymfony/FOSHttpCacheBundle/blob/19e8057bf0f3abe67ecb9cd2777895b590196692/.travis.yml#L51

the lowest version is probably that the monolog-bridge version 2.7 gets installed, according to the composer output, while we have the http-kernel at 3.4 and it does not have the logger interface. maybe we need to mark a conflict with monolog-brige < 3.0?

**Comment by Toflar** [comment]
Done but still... :)

**Comment by dbu** [comment]
`Downgrading symfony/monolog-bridge (v4.1.4 => v2.7.0): Loading from cache`

> the lowest version is probably that the monolog-bridge version 2.7 gets installed, according to the composer output, while we have the http-kernel at 3.4 and it does not have the logger interface. maybe we need to mark a conflict with monolog-brige < 3.0?

File: composer.json [hunk 1]
```diff
@@ -52,7 +52,8 @@
         "symfony/console": "To send invalidation requests from the command line"
     },
     "conflict": {
-        "twig/twig": "<1.12.0"
+        "twig/twig": "<1.12.0",
+        "symfony/monolog-bridge": "<3.0"
```
Comment (dbu) [inline composer.json]: can we use 3.4 here to be in sync with the other symfony components?

**Comment by dbu** [comment]
thanks!

**Pull request merged**